packages/synapse-react-client/src/components/DataGrid/DataGrid.tsx

```diff
@@ -441,14 +442,17 @@ const DataGrid = () => {
           margin: '10px 0',
           padding: '10px',
           border: '1px solid #ccc',
-          maxHeight: '200px',
+          maxHeight: '400px',
           overflowY: 'auto',
         }}
       >
         <h3>Model snapshot</h3>
-        <p>
-          {modelSnapshot ? JSON.stringify(modelSnapshot) : 'No model available'}
-        </p>
+
+        {modelSnapshot ? (
+          <ComplexJSONRenderer value={modelSnapshot} />
```

Comment:
Interactive component for looking at JSON:

<img width="1329" height="427" alt="Image" src="https://github.com/user-attachments/assets/7a88eb0e-2556-48e1-8a74-17ed9f9878d3" />